open_gpt/models/loading.py

```diff
@@ -7,6 +7,7 @@
 
 def load_model_and_tokenizer(
```

Comment:
I would suggest to have a mixin function `load_lora(self)` which can be used by different model wrapper without copy codes.

Reply:
how about using a decorator `@load_lora()` to do this?